### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by inverse**
- Add `.nvmrc`
- Flip checkout and reading `.nvmrc` shouldnt the code be checked out first?
- run `npm audit fix` p.s. I enabled github security alerts too here
- Fix typo + validation

### CODE REVIEW

File: scripts/dump-connectors.js [hunk 1]
```diff
@@ -20,10 +20,15 @@ const listFile = `${resDir}/connectors.json`;
 
 async function main(args) {
 	const latestTag = args[2];
-	let exitCode = 0;
 
+	if (latestTag === undefined) {
```
Comment (alexesprit) [inline scripts/dump-connectors.js:24]: It could be simplified to `if (!latestTag)`.

Comment (inverse) [inline scripts/dump-connectors.js:24]:
```suggestion
	if (!latestTag) {
```

Comment (inverse) [reply scripts/dump-connectors.js]: thnx :)

**Review by alexesprit — APPROVED** [review]

File: .github/workflows/resources.yml [hunk 2]
```diff
@@ -7,10 +7,10 @@ jobs:
     env:
       VERSION: ${{ github.event.action }}
     steps:
+    - uses: actions/checkout@v2
     - name: Read .nvmrc
       run: echo ::set-output name=nvmrc::$(cat .nvmrc)
       id: nvm
-    - uses: actions/checkout@v2
```
Comment (alexesprit) [inline .github/workflows/resources.yml:13]: Haha, good catch! I also found the same issue in the workflow from `web-scrobbler.github.io` (looks like I copied that workflow from that repo). :)

**Pull request merged**